app/src/pages/inside/logsPage/historyLine/historyLine.jsx

```diff
@@ -66,6 +78,29 @@ export class HistoryLine extends Component {
       <div className={cx('history-line')}>
         <ScrollWrapper autoHeight hideTracksWhenNotNeeded>
           <div className={cx('history-line-items')}>
+            {historyItems.length === DEFAULT_HISTORY_DEPTH && (
+              <GhostButton
+                style={{
```

Comment:
Most likely you need to reuse one of the existing button components. Overwise you should create the new one. I suggest to research if the needed component exists, overwise discuss with the designer are you really need a new one.
Never use inline styles unnecessarily.